packages/vue/src/components/atoms/SfImage/SfImage.vue

```diff
@@ -140,6 +152,31 @@ export default {
             : `${this.height}px`,
       };
     },
+    imageComponentTag() {
+      return !this.$nuxt ? "img" : this.imageTag;
+    },
+    conditionalFit() {
+      const fitCheck = () => {
+        if (this.nuxtImgConfig.fit) {
```

Comment:
suggestion: I think it can be simplified with ``` if (!this.nuxtImgConfig.fit) { console.error("Missing prop fit") } return ...```

Reply:
of course